.vscode/launch.json

```diff
@@ -0,0 +1,14 @@
+{
+    // Use IntelliSense to learn about possible attributes.
+    // Hover to view descriptions of existing attributes.
+    // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
+    "version": "0.2.0",
```

Comment:
Strictly speaking, .json files aren't allowed to have comments. That's why GitHub is marking these in red.

https://stackoverflow.com/questions/244777/can-comments-be-used-in-json

May not be a big deal because this looks like a VS Code file, but just wanted to note the standard.

Reply:
Is the JSON file ok to keep? Or should I remove it from the PR?

Reply:
Yeaaahhh you know, this looks more like a local configuration that was specific to your workflow. So technically not part of the code base, but an individual developer's specific settings, which might vary.

Yes, let's remove this, and add `.vscode` to _.gitignore_ so that people are free to configure VS Code the way they choose.

Reply:
Done!